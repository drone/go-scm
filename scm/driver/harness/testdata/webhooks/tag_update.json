{
  "trigger": "tag_updated",
  "repo": {
    "id": 16,
    "path": "kmpySmUISimoRrJL6NL73w/harness-core",
    "identifier": "harness-core",
    "default_branch": "develop",
    "git_url": "http://localhost:3000/git/kmpySmUISimoRrJL6NL73w/harness-core.git",
    "uid": "harness-core"
  },
  "principal": {
    "id": 2,
    "uid": "lv0euRhKRCyiXWzS7pOg6g",
    "display_name": "Admin",
    "email": "admin@harness.io",
    "type": "user",
    "created": 1701091219051,
    "updated": 1701091219051
  },
  "ref": {
    "name": "refs/tags/ddxas",
    "repo": {
      "id": 16,
      "path": "kmpySmUISimoRrJL6NL73w/harness-core",
      "identifier": "harness-core",
      "default_branch": "develop",
      "git_url": "http://localhost:3000/git/kmpySmUISimoRrJL6NL73w/harness-core.git",
      "uid": "harness-core"
    }
  },
  "sha": "700b3dab8e7a5cebf5e1ce54e7dd5bde60099912",
  "head_commit": {
    "sha": "700b3dab8e7a5cebf5e1ce54e7dd5bde60099912",
    "message": "Asd",
    "author": {
      "identity": {
        "name": "Abhinav Singh",
        "email": "abhinav.singh@harness.io"
      },
      "when": "2024-03-07T03:18:51-08:00"
    },
    "committer": {
      "identity": {
        "name": "Abhinav Singh",
        "email": "abhinav.singh@harness.io"
      },
      "when": "2024-03-07T03:18:51-08:00"
    },
    "added": [],
    "removed": [],
    "modified": []
  },
  "commits": [
    {
      "sha": "700b3dab8e7a5cebf5e1ce54e7dd5bde60099912",
      "message": "Asd",
      "author": {
        "identity": {
          "name": "Abhinav Singh",
          "email": "abhinav.singh@harness.io"
        },
        "when": "2024-03-07T03:18:51-08:00"
      },
      "committer": {
        "identity": {
          "name": "Abhinav Singh",
          "email": "abhinav.singh@harness.io"
        },
        "when": "2024-03-07T03:18:51-08:00"
      },
      "added": [],
      "removed": [],
      "modified": []
    }
  ],
  "total_commits_count": 1,
  "commit": {
    "sha": "700b3dab8e7a5cebf5e1ce54e7dd5bde60099912",
    "message": "Asd",
    "author": {
      "identity": {
        "name": "Abhinav Singh",
        "email": "abhinav.singh@harness.io"
      },
      "when": "2024-03-07T03:18:51-08:00"
    },
    "committer": {
      "identity": {
        "name": "Abhinav Singh",
        "email": "abhinav.singh@harness.io"
      },
      "when": "2024-03-07T03:18:51-08:00"
    },
    "added": [],
    "removed": [],
    "modified": []
  },
  "old_sha": "0f1835abe08473e07863540712d8389984b72dad",
  "forced": true
}