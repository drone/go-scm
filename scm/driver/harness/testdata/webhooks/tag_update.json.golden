{
  "Ref": "refs/tags/ddxas",
  "Before": "0f1835abe08473e07863540712d8389984b72dad",
  "After": "700b3dab8e7a5cebf5e1ce54e7dd5bde60099912",
  "Repo": {
    "ID": "16",
    "Namespace": "",
    "Name": "harness-core",
    "Perm": null,
    "Branch": "develop",
    "Private": false,
    "Clone": "http://localhost:3000/git/kmpySmUISimoRrJL6NL73w/harness-core.git",
    "CloneSSH": "",
    "Link": "http://localhost:3000/git/kmpySmUISimoRrJL6NL73w/harness-core.git",
    "Created": "0001-01-01T00:00:00Z",
    "Updated": "0001-01-01T00:00:00Z"
  },
  "Commit": {
    "Sha": "700b3dab8e7a5cebf5e1ce54e7dd5bde60099912",
    "Message": "Asd",
    "Author": {
      "Name": "Abhinav Singh",
      "Email": "abhinav.singh@harness.io",
      "Date": "0001-01-01T00:00:00Z",
      "Login": "",
      "Avatar": ""
    },
    "Committer": {
      "Name": "Abhinav Singh",
      "Email": "abhinav.singh@harness.io",
      "Date": "0001-01-01T00:00:00Z",
      "Login": "",
      "Avatar": ""
    },
    "Link": ""
  },
  "Commits": [
    {
      "Sha": "700b3dab8e7a5cebf5e1ce54e7dd5bde60099912",
      "Message": "Asd",
      "Author": {
        "Name": "Abhinav Singh",
        "Email": "abhinav.singh@harness.io",
        "Date": "0001-01-01T00:00:00Z",
        "Login": "",
        "Avatar": ""
      },
      "Committer": {
        "Name": "Abhinav Singh",
        "Email": "abhinav.singh@harness.io",
        "Date": "0001-01-01T00:00:00Z",
        "Login": "",
        "Avatar": ""
      },
      "Link": ""
    }
  ],
  "Sender": {
    "ID": "lv0euRhKRCyiXWzS7pOg6g",
    "Login": "lv0euRhKRCyiXWzS7pOg6g",
    "Name": "Admin",
    "Email": "admin@harness.io",
    "Avatar": "",
    "Created": "2023-11-27T05:20:19.051-08:00",
    "Updated": "2023-11-27T05:20:19.051-08:00"
  }
}